src/astra_driver.cpp

```diff
@@ -158,7 +158,7 @@ void AstraDriver::advertiseROSTopics()
 {
   rmw_qos_profile_t custom_camera_qos_profile = rmw_qos_profile_default;
 
-  custom_camera_qos_profile.depth = 1;
+  custom_camera_qos_profile.depth = 50;
```

Comment:
tested on the tb2 here we had pretty good results with the [default sensor profile ](https://github.com/ros2/rmw/blob/7bc9eec72c39641105424114fd436990d17e7b4e/rmw/include/rmw/qos_profiles.h#L28)(for all big data streams aka Depth image, pointcloud and laserscans). Do you have an idea of the difference in performance between 5 and 50 ? A queue size of 50 seems huge for data as large as images